clarity/src/vm/analysis/analysis_db.rs

```diff
@@ -130,9 +130,9 @@ impl<'a> AnalysisDatabase<'a> {
                     .map_err(|_| CheckErrors::Expects("Bad data deserialized from DB".into()))
             })
             .transpose()?
-            .and_then(|mut x| {
+            .map(|mut x| {
```

Comment:
Can this be `inspect()` to avoid returning a value?

Reply:
Unfortunately not in this case because I mutate x and inspect doesn't let you mutate the inner value.